be/src/olap/cumulative_compaction.cpp

```diff
@@ -117,7 +117,7 @@ Status CumulativeCompaction::pick_rowsets_to_compact() {
     // candidate_rowsets may not be continuous
     // So we need to choose the longest continuous path from it.
     std::vector<Version> missing_versions;
-    RETURN_NOT_OK(find_longest_consecutive_version(&candidate_rowsets, &missing_versions));
+    OLAP_RETURN_NOT_OK(find_longest_consecutive_version(&candidate_rowsets, &missing_versions));
```

Comment:
Do not add this new macro. If RETURN_NOT_OK is conflict with arrow's macro definition. You could remove this macro "RETURN_NOT_OK" totally and use RETURN_IF_ERROR instead. They are same.